server/firestore.rules

```diff
@@ -0,0 +1,8 @@
+rules_version = '2';
+service cloud.firestore {
+  match /databases/{database}/documents {
+    match /{document=**} {
+      allow read, write: if true;
```

Comment:
For future ref can you just make a comment about why we need this and what to change if we want to revert.